voctogui/lib/studioclock.py

```diff
@@ -12,26 +12,27 @@ class StudioClock(Gtk.ToolItem):
 
     # set resolution of the update timer in seconds
     timer_resolution = 0.1
+    time = time.localtime(0)
```

Comment:
I think I would rather name this Variable `start_time`or stf. `time` is quite generic and might collide with the `time` module.